en/guide/toolchain.md

```diff
@@ -0,0 +1,121 @@
+```cmake
+if(WINDOWS)
+    set(dronecore_lib "${CMAKE_SOURCE_DIR}/../../install/lib/dronecore.lib")
+else()
+    set(dronecore_lib "${CMAKE_SOURCE_DIR}/../../install/lib/libdronecore.a")
+endif()
+```
+
+The final section of the file defines the libraries to link against if DroneCore is installed globally.
+```cmake
+target_link_libraries(takeoff_and_land
+    ${dronecore_lib}
+    # If installed system-wide:
+    # dronecore
+    ${CMAKE_THREAD_LIBS_INIT}
+    ${platform_libs}
+)
+```
+
+> **Tip** Getting relative paths right can be difficult. You can use [message()](https://cmake.org/cmake/help/v3.9/command/message.html) to print out debug information in your build:
+```bash
+# Get the location of your CMakeLists.txt file
+message(${CMAKE_SOURCE_DIR})
+```
+
+## Building the App
+
+This section assumes that you have already [built and installed the DroneCore C++ Library](../contributing/build.md) and that your example has a **CMakeLists.txt** in its root directory. 
+
+All we need to do is create a build directory (to keep all our build files in one place) and run the build from there. 
+
+```bash 
+cd /path/to/your/example/  # Open a prompt and navigate to your example 
+mkdir build && cd build   # Create a build directory and navigate into it
+cmake --build .  # Build the current directory
+./your_executable_name  # Run your new executable
+```
+
+
+## Useful Links
+
+We only show a small fraction of what *Cmake* is capable of! Check out the links below for more information and examples.
+
+* [cmake resources page](https://cmake.org/documentation/)
+* [cmake documentation](https://cmake.org/cmake/help/latest/) (latest)
+* [cmake commands](https://cmake.org/cmake/help/latest/manual/cmake-commands.7.html) (used in this example):
+  * [cmake_minimum_required()](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html) - Minimum compatible version of cmake
+  * [project()](https://cmake.org/cmake/help/latest/command/project.html) - Set a name for project (and optionally version, languages, etc.)
+  * [add_definitions()](https://cmake.org/cmake/help/latest/command/add_definitions.html) - Adds -D define flags to the compilation of source files
+  * [set()](https://cmake.org/cmake/help/latest/command/set.html) - Set a normal, cache, or environment variable to a given value
+  * [find_package()](https://cmake.org/cmake/help/latest/command/find_package.html) - Load settings for an external project.
+  * [include_directories()](https://cmake.org/cmake/help/latest/command/include_directories.html)
+  * [add_executable()](https://cmake.org/cmake/help/latest/command/add_executable.html) - Specify executable to be created and set of source files to use
+  * [target_link_libraries()](https://cmake.org/cmake/help/latest/command/target_link_libraries.html) - Specify libraries to use when linking
+  * [message()](https://cmake.org/cmake/help/v3.9/command/message.html) - Display a message to the user
```

Comment:
Here you link to 3.9, in the others to latest. For my taste, that's almost too many links. I would probably slim it down a bit.

Reply:
OK. I'll change everything to latest. I'll move the links into the body of the doc and leave just the main entry point links here. Thanks.